api/providers/serializers.py

```diff
@@ -203,6 +203,7 @@ class Meta:
         'share_publish_type',
         'reviews_workflow',
         'permissions',
+        'advertise_on_discover_page',
```

Comment:
You've made this filterable, which is good, but you still have to add the field to the serializer to get it serialized in the API response. Somewhere around L213 below.